packages/core-web/src/index.js

```diff
@@ -83,7 +83,25 @@ function detectAndApplyFixedHeaderStyles() {
     });
     resizeObserver.observe(headerSelector[0]);
   };
+
+  let lastOffset = 0;
+  const toggleHeaderOnScroll = () => {
+    // prevent toggling of header on desktop site
+    if (window.innerWidth > 767) { return; }
+    const currentOffset = window.pageYOffset;
+    const isEndOfPage = (window.innerHeight + currentOffset) >= document.body.offsetHeight;
+    // to prevent page from auto scrolling when header is toggled at the end of page
+    if (isEndOfPage) { return; }
+    if (currentOffset > lastOffset) {
+      headerSelector.addClass('hide-header');
+    } else {
+      headerSelector.removeClass('hide-header');
+    }
+    lastOffset = currentOffset;
+  };
+
   addResizeHeaderListener();
```

Comment:
could we prevent this from running when caused by the changes in `max-height`?

so we don't observe the secondary movements of the content when hiding / showing the navbar

![navbar-hiding](https://user-images.githubusercontent.com/3306138/107840810-d222b280-6df0-11eb-9e72-5c925a518e2a.gif)

Reply:
Alright updated :)